examples/pytorch/BertNewsClassification/bert_classification.py

```diff
@@ -235,8 +236,7 @@ def __init__(self, **kwargs):
             param.requires_grad = False
         self.drop = nn.Dropout(p=0.2)
         # assigning labels
-        self.class_names = ["world", "Sports", "Business", "Sci/Tech"]
-        n_classes = len(self.class_names)
+        n_classes = 4
```

Comment:
I noticed `self.class_names` is defiend just to calculate the number of classes so I removed it.